src/jws.rs

```diff
@@ -2,1451 +2,8 @@
 //!
 //! Defined in [RFC 7515](https://tools.ietf.org/html/rfc7515). For most common use,
 //! you will want to look at the  [`Compact`](enum.Compact.html) enum.
-use std::str;
-use std::sync::Arc;
+mod compact;
+mod flattened;
```

Comment:
Let's make these two modules `pub`. Please move the relevant compact JWS documentation to the module documentation of `compact` and we can leave a general documentation about JWS in this module.

We can leave the `pub use` below to maintain some compatibility.